apps/frontend/src/pages/auth/components/auth-layout/auth-layout.tsx

```diff
@@ -0,0 +1,24 @@
+import "./auth-layout.css";
+
+interface AuthLayoutProperties {
+	children: React.ReactNode;
+}
```

Comment:
```suggestion
type Properties {
	children: React.ReactNode;
}
```
Use same approach everywhere